src/language/dafnyInstallation.ts

```diff
@@ -65,14 +81,14 @@ export class DafnyInstaller {
     private readonly statusOutput: OutputChannel
   ) {}
 
-  public static isMinimumRequiredLanguageServer(version: string): boolean {
+  public static isLatestKnownLanguageServerOrNewer(version: string): boolean {
     if(version === LanguageServerConstants.UnknownVersion) {
       return true;
     }
     const [ givenMajor, givenMinor ] = version.split('.');
-    const [ requiredMajor, requiredMinor ] = LanguageServerConstants.RequiredVersion.split('.');
-    return givenMajor > requiredMajor
-      || givenMajor === requiredMajor && givenMinor >= requiredMinor;
+    const [ latestRequired, latestMinor ] = LanguageServerConstants.LatestVersion.split('.');
+    return givenMajor > latestRequired
+      || givenMajor === latestRequired && givenMinor >= latestMinor;
```

Comment:
Shouldn't this have three components?

Reply:
Something like this:

```
  public static isMinimumRequiredLanguageServer(version: string): boolean {
      const v = version.split('.');
      const req = LanguageServerConstants.MinVersion.split('.');
      for (var i = 0; i < req.length; i++) {
          var vi = v[i] ?? "0";
          var ri = req[i];
          if (vi == ri)
              continue
          return vi > ri;
      }
      return true;
  }
```

Reply:
You're right, it's probably best to include the patch part of the version string too. I can't really remember the reasoning behind excluding it anymore, or it might no longer apply.